index.html

```diff
@@ -5132,6 +5132,9 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
```

Comment:
```suggestion
  <p>
    This media type MUST NOT be used to describe a verifiable credential with an <dfn>embedded proof</dfn>.
  </p>
  <p>
```

Reply:
We have similar text with strong support on https://github.com/w3c/vc-data-model/pull/1014

I created this PR assuming @dlongley and @msporny might accept it if this text was not present.

Reply:
I won't be accepting this suggestion.